core/src/test/scala/unit/kafka/utils/SchedulerTest.scala

```diff
@@ -87,18 +87,29 @@ class SchedulerTest {
     assertEquals(1, counter1.get, "Should only run once")
   }
 
+  @Test
+  def testNonPeriodicTaskWhenPeriodEqual0(): Unit = {
```

Comment:
We usually don't have number in test name, could we rename to `testNonPeriodicTaskWhenPeriodIsZero`

Reply:
> We usually don't have number in test name, could we rename to `testNonPeriodicTaskWhenPeriodIsZero`

It has been completed.Thanks